src/components/PeopleList.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Person } from '../types/Person';
+
+type Props = {
+  people: Person[],
+  onSelected: (person: Person) => void,
+  setFocus: (boolean: boolean) => void
+  setQuery: (person: string) => void
+};
+
+export const PeopleList: React.FC<Props> = React.memo(({
+  people,
+  onSelected = () => {},
+  setFocus = () => {},
+  setQuery = () => {},
+}) => {
+  return (
+    <div className="dropdown-menu" role="menu" id="dropdown-menu">
+      <div className="dropdown-content">
+
+        {people.map(person => (
```

Comment:
```suggestion
      <div className="dropdown-content">
        {people.map(person => (
```
Don't leave empty lines between parent and child elements